tests/test_bake_project.py

```diff
@@ -181,7 +181,7 @@ def check_quality(result):
         pylintrc = str(result.project.join('pylintrc'))
         for filename in filenames:
             name = os.path.join(dirpath, filename)
-            if not name.endswith('.py'):
+            if not name.endswith('.py') or filename == 'conf.py':
```

Comment:
I'm afraid I don't understand enough of the intricacies of how this test works to know why you have to exclude `conf.py` now.

Can you explain any further what the problem is?